test/unit/3p/test-3p-frame.js

```diff
@@ -42,8 +43,11 @@ describes.realWin('3p-frame', {amp: true}, (env) => {
     document = window.document;
   });
 
-  function mockMode(mode) {
-    env.sandbox.stub(window.parent, '__AMP_MODE').value(mode);
+  function mockMode(options) {
+    env.sandbox.stub(window.parent, '__AMP_MODE').value(options);
```

Comment:
Is this line still necessary?

Reply:
This is the one call where removing the line breaks the test case: https://github.com/ampproject/amphtml/blob/5ece4457d5b70035a4135ece9306416866538e0f/test/unit/3p/test-3p-frame.js#L153-L158

It looks like this is specifically because the `mode` changes when not mocked via window: https://github.com/ampproject/amphtml/blob/7df04fada6e2a5857ff8b172f6d380db6c51bd0b/src/iframe-attributes.js#L90

I propose we leave the stub for now, and in a separate PR, see about updating `getModeObject` to reference `#core/mode` over `getMode(win)`: https://github.com/ampproject/amphtml/blob/7df04fada6e2a5857ff8b172f6d380db6c51bd0b/src/mode-object.js#L25-L33